README.md

```diff
@@ -20,31 +20,20 @@ You may install additional [backend extensions](https://docs.mopidy.com/en/lates
 ### Usage
 
 #### PulseAudio over network
+    
+    Mount the current user's pulse directory to the pulseuadio user (id - 105)
+    Based on: https://github.com/TheBiggerGuy/docker-pulseaudio-example
 
-First to make [audio from from within a Docker container](http://stackoverflow.com/q/28985714/167897), you should enable [PulseAudio over network](https://wiki.freedesktop.org/www/Software/PulseAudio/Documentation/User/Network/); so if you have X11 you may for example do:
-
- 1. Install [PulseAudio Preferences](http://freedesktop.org/software/pulseaudio/paprefs/). Debian/Ubuntu users can do this:
-
-        $ sudo apt-get install paprefs
-
- 2. Launch `paprefs` (PulseAudio Preferences) > "*Network Server*" tab > Check "*Enable network access to local sound devices*" (you may check "*Don't require authentication*" to avoid mounting cookie file described below).
-
- 3. Restart PulseAudio
-
-        $ sudo service pulseaudio restart
-
-    or
-
-        $ pulseaudio -k
-        $ pulseaudio --start
+    $ docker run -d \
+      -v /run/user/$(id -u)/pulse:/run/user/105/pulse \
```

Comment:
Sorry for the delay.

- On my host machine, there is no user with id 105, `-v /run/user/$(id -u)` this might work universally (This uses the current user's pulse socket) 

- Unfortunately I have no idea why,  I tried all the variations, the only one that works is when I create a new user...